### PR TITLE
fix: Unchecked Casts

### LINKED ISSUE #1253: Fix unchecked Casts
**Describe the chore**
Currently the following warning is being shown in OrdersUnderUserViewModel class:-

Unchecked cast: List<Int?> to kotlin.collections.ArrayList<Int> /* = java.util.ArrayList<Int> */

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by somenath1435**
Fixes #1253 

Changes: Fixed unchecked casts.

### CODE REVIEW

**Comment by somenath1435** [comment]
@nikit19 But I did fixed the warning

**Comment by haroldadmin** [comment]
@somenath1435 No, you supressed the warning. The underlying problem still exists. Maybe you could try looking into Kotlin's safe cast operator (`as?`).

**Comment by iamareebjamal** [comment]
Man, you just killed the patient and claiming to cure the cancer

**Comment by liveHarshit** [comment]
Can you please point, for which cast you are talking about?

**Comment by somenath1435** [comment]
@liveHarshit This warning comes:
Unchecked cast: List<Int?> to kotlin.collections.ArrayList

**Comment by liveHarshit** [comment]
> @liveHarshit This warning comes:
> Unchecked cast: List<Int?> to kotlin.collections.ArrayList

Then use smart cast instead. Suppress mean you just ignore the warning.

**Comment by somenath1435** [comment]
@liveHarshit Can you please explain?

**Comment by liveHarshit** [comment]
Learn about Kotlin smart cast if it works .

**Comment by somenath1435** [comment]
I've researched a little.
Looks like "as?" does the job.
What's your opinion @liveHarshit ?

**Comment by somenath1435** [comment]
@iamareebjamal please help

**Comment by iamareebjamal** [comment]
Change `MutableLiveData<ArrayList<Int>>()` ArrayList to List<Int>

Don't know why people are so in love with using ArrayList

**Comment by iamareebjamal** [comment]
I literally mentioned the fix. All you have to do is remove 5 characters in a line. Updates?

This PR is open to all if there are no changes by tomorrow 

/remind me make these changes myself in 2 days

**Comment by iamareebjamal** [comment]
Closing PRs and creating new ones is a perfect recipe for not getting them reviewed.

File: app/src/main/java/org/fossasia/openevent/general/order/Order.kt [hunk 1]
```diff
@@ -47,5 +47,5 @@ data class Order(
     @Relationship("event")
     var event: EventId? = null,
     @Relationship("attendees")
-    var attendees: List<AttendeeId>? = null
+    var attendees: ArrayList<AttendeeId>? = null
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/order/Order.kt]: You made exactly opposite change in a file which has no relation to the attendee view model

**Pull request closed without merging**